pkgs/applications/misc/blender/2.93.nix

```diff
@@ -0,0 +1,165 @@
+{ stdenv, lib, config, fetchzip, cmake, makeWrapper
+# Common:
+, alembic, boost, embree, ffmpeg, fftw, freetype, gettext
+, glew, gmp, ilmbase, jemalloc, libharu, libjpeg, libpng
+, libsamplerate, libsndfile, libtiff, opencolorio, openexr
+, openimagedenoise, openimageio2, openjpeg, opensubdiv
+, potrace, pugixml, python3, tbb, zlib
+
+# Linux specific:
+, libGL, libGLU, libX11, libXext, libXi, libXrender, libXxf86vm
+, openxr-loader, openvdb, openal, ocl-icd
```

Comment:
```suggestion
, openxr-loader, openvdb, openal
```

Reply:
Note that this should be in `default.nix` instead of `2.93.nix` i assume.

Reply:
Right